integrations/llamaindex/vector-stores/queryengine-tool.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
we should have pinecone and simplestore nodes

Reply:
After updates, we need to update SUMMARY.md as well

Reply:
done